lms/djangoapps/lti_provider/management/commands/rescore_lti.py

```diff
@@ -0,0 +1,45 @@
+"""
+Management command to rescore all lti problems for the given course for all users.
+"""
+
+import textwrap
+
+from django.core.management import BaseCommand
+
+from opaque_keys.edx.keys import CourseKey
+
+from lti_provider.models import GradedAssignment
+from lti_provider import tasks
+from student.models import CourseEnrollment
+
+
+class Command(BaseCommand):
+    """
+    Rescore all lti problems for the given courses for all users.
```

Comment:
Nit: as mentioned above, we're not "rescoring" anything, just firing notice of prior scoring.